### DISCUSSION TITLE
英文文档翻译官招募中

### DISCUSSION THREAD

**Discussion by janryWang** [alibaba/formily#1205] [Ideas]
受限于我自身的英文水平，欢迎壮士贡献进来帮忙翻译英文版文档，本人在此跪谢

**Comment by aspirantzhang** [comment]
我看umi都是谷歌机翻，要不咱也...🤣

**Reply by MatriXiao88** [reply]
也可以试试这个，稍微更加准确一点点：https://www.deepl.com/translator

**Comment by liuweiGL** [comment]
现在官网默认的是英文，请问中文文档在哪呢？

**Reply by janryWang** [reply]
右上角按钮可以切换啊

**Reply by liuweiGL** [reply]
嗷，我台式机看到了切换按钮，昨天用超极本浏览的时候没显示。
